### ISSUE TITLE
Accessibility: Unable to select chart menus items on dashboards 

### ISSUE THREAD

**Issue by yousoph** [apache/superset#26217]
#### How to reproduce the bug

1. Create a dashboard with a few charts
2. Use the keyboard navigation to navigate to the ... menu at the top right of charts 

### Expected results
User is unable to select menu items 
<img width="368" alt="image" src="https://github.com/apache/superset/assets/10627051/6114cec8-f7ab-420b-bd75-f446a31f3890">

https://www.w3.org/TR/WCAG22/#keyboard

### Actual results

User is able to select menu items using keyboard navigation

### Environment

(please complete the following information):

- browser type and version:
- superset version: `master`
- python version: `python --version`
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by rusackas** [comment]
Is anyone concerned with this still? Tempted to close as stale. Maybe a bounty candidate, but open to a PR from anyone reading this.

**Comment by geido** [comment]
This should be working as we complete the migration to Ant Design 5. Closing it for now.
